inference-engine/samples/benchmark_app/CMakeLists.txt

```diff
@@ -8,5 +8,6 @@ file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
 ie_add_sample(NAME benchmark_app
               SOURCES ${SRC}
               HEADERS ${HDR}
-              DEPENDENCIES format_reader
+              DEPENDENCIES format_reader samples_utils
```

Comment:
let's rename `samples_utils` to `ie_samples_utils`

Reply:
Renamed.